avivator/src/utils.js

```diff
@@ -65,7 +65,7 @@ export async function createLoader(
       // Non-Bioformats6 pyramids use Image tags for pyramid levels and do not have offsets
       // built in to the format for them, hence the ternary.
       const {
-        omexml: { SizeZ, SizeT, SizeC },
```

Comment:
This is the pattern I'm talking about below that we can move away from. We have a utility that creates a loader and then we derive these properties immediately from the loader for use in the UI. I'm ok with this for right now, but let's think about moving metadata off of the loader.

Instead  our utility could be along the lines: `openTIFF(url) -> { loader, metadata }`